build.sbt

```diff
@@ -48,6 +48,14 @@ lazy val commonJacocoReportSettings: JacocoReportSettings = JacocoReportSettings
 lazy val commonJacocoExcludes: Seq[String] = Seq(
 )
 
+enablePlugins(FlywayPlugin)
+flywayUrl := FlywayConfiguration.flywayUrl
+flywayUser := FlywayConfiguration.flywayUser
+flywayPassword := FlywayConfiguration.flywayPassword
+flywayLocations := FlywayConfiguration.flywayLocations
+flywaySqlMigrationSuffixes := FlywayConfiguration.flywaySqlMigrationSuffixes
+libraryDependencies ++= flywayDependencies
```

Comment:
I think it adds to the global dependencies, doesn't it? I think flyway dependencies shouldn't be a generic dependency.

Reply:
I believe this is what @salamonpavel was dealing with in the past and there was no way how to circumvent this (I know :( ) by extracting / moving elsewhere. Anyway, it's part of the build, i.e. general dependency: https://github.com/AbsaOSS/atum-service/blob/master/build.sbt#L53-L59

(remember, Flyway is temporary anyway)

Reply:
I agree with @benedeki 

```sbt
lazy val modulNeedingFlyaway = project
  .enablePlugins(FlywayPlugin)
  .settings(
    flywayUrl := FlywayConfiguration.flywayUrl
    flywayUser := FlywayConfiguration.flywayUser
    flywayPassword := FlywayConfiguration.flywayPassword
    flywayLocations := FlywayConfiguration.flywayLocations
    flywaySqlMigrationSuffixes := FlywayConfiguration.flywaySqlMigrationSuffixes
    libraryDependencies ++= flywayDependencies
  )
```

and then run it like `sbt modulNeedingFlyaway/flywayMigrate`

Reply:
Can we redefine the `flywayMigrate` to behave like `sbt modulNeedingFlyaway/flywayMigrate`?

Reply:
Like this `addCommandAlias("flywayMigrate", "modulNeedingFlyaway/flywayMigrate")`?

Reply:
Let me try, thanks @Zejnilovic. To be fair I haven't been playing with this one as my primary objective here was to actually have a IT suite embedded in our CI - but I agree that we can improve this mechanism in build.sbt

Reply:
That's why it got approved, not blocking the whole thing 😉